Comp2023/src/main/java/frc/robot/commands/AutoPreloadHigh.java

```diff
@@ -42,11 +43,13 @@ public AutoPreloadHigh(Elbow elbow, Extension extension, Wrist wrist, Gripper gr
         new PrintCommand(getName() + ": Move Extension for Preload"),   
         new ExtensionMoveToLength(extension, ExtensionLength.EXTENSION_HIGH),
 
-        new PrintCommand(getName() + ": Move Wrist for Preload"),   
-        new WristMoveToAngle(wrist, WristAngle.WRIST_SCORE),
+        new ParallelDeadlineGroup(

```

Comment:
This should probably be a simple sequential set of commands that ensures the gripper runs in EXPEL mode while the extension is retracted. Since the gripper can be turned while other commands run, this provides the parallelism.

Here's very rough pseudo-code.

"Move Wrist to Score Preload"
WristMoveToAngle(wrist, WristAngle.WRIST_SCORE)

"Turn on gripper to expel Preload"
GripperRun(gripper, GRConsts.GRMode.GR_EXPEL)

"Retract extension to MID level which is easier on extension motor"
ExtensionMoveToLength(extension, ExtensionLength.EXTENSION_MID)

"Turn off gripper because the extension is already retracted"
GripperRun(gripper, GRConsts.GRMode.GR_STOP)